docs/nodes/credentials/SurveyMonkey/README.md

```diff
@@ -18,4 +29,4 @@ Create a [SurveyMonkey](https://surveymonkey.com/) account.
 7. Once created, go to settings.
 8. Use the access token, client ID and client secret with your SurveyMonkey node credentials in n8n.
 
-![Getting SurveyMonkey credentials](./using-access-token.gif)
+![Getting SurveyMonkey credentials](./getting-credentials.gif)
```

Comment:
I am not sure why this rename was necessary here?

Reply:
The GIF is the same for OAuth and Access Token credentials; thought it might be more appropriate to rename it